backend/src/main/java/com/redhat/cloud/notifications/models/EmailSubscriptionProperties.java

```diff
@@ -1,4 +1,22 @@
 package com.redhat.cloud.notifications.models;
 
 public class EmailSubscriptionProperties extends EndpointProperties {
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (!(o instanceof EndpointProperties)) {
+            return false;
+        }
+
+        return true;
```

Comment:
This is a placeholder when we add more properties.

Reply:
Not really, this could be part of this PR or the next PR that actually adds attributes to this class.

Reply:
I'm OK with having a placeholder here, that's not why I commented this. My concern is about the compatibility between the existing `EndpointProperties` `equals/hashCode` implementation and the future implementatations based on `EmailSubscriptionProperties` attributes.

Reply:
The issue here is that for the `getOrCreate` we won't have any `id` on the received properties, so we will need to compare the attributes instead.

Reply:
I'm resolving this since there's another discussion about the same subject. To be continued 😄